frontends/election-manager/src/app.test.tsx

```diff
@@ -373,7 +373,7 @@ test('printing ballots, print report, and test decks', async () => {
     expect.stringContaining(LogEventId.PrintedBallotReportPrinted)
```

Comment:
Is there an L&A workflow that we're building that could use its dedicated test? or does that test already exist somewhere?

Reply:
There's no test yet for an L&A workflow, but I'm planning to add one once I've further built the L&A page out. For now, I just wanted to make sure that test-ballot functionality and zero-report functionality were working in isolation.

Reply:
Actually, on second thought, it doesn't hurt to start building the test for that flow out now, even if the flow isn't complete yet. Will add!

Reply:
Done! https://github.com/votingworks/vxsuite/pull/1708/commits/7a6f3a7090f61252b43fa08def6e020b0a41a22e

Note that I removed L&A testing from the existing `printing ballots, print report, and test decks` test, renaming it `printing ballots and printed ballots report`, since we now have the standalone test for L&A